docs/migration_ov_2_0/snippets/ov_api/ov_common.cpp

```diff
@@ -0,0 +1,34 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+#include <openvino/core/function.hpp>
+#include <openvino/runtime/runtime.hpp>
+
+int main() {
+    //! [ov_api_2_0:create_core]
+    ov::runtime::Core core;
+    //! [ov_api_2_0:create_core]
+
+    //! [ov_api_2_0:read_model]
+    std::shared_ptr<ov::Function> network = core.read_model("model.xml");
+    //! [ov_api_2_0:read_model]
+
+    //! [ov_api_2_0:get_inputs_outputs]
+    ngraph::ParameterVector inputs = network->get_parameters();
```

Comment:
should it be op::ParameterVector?

Reply:
I have been written this example before nGraph migration to OV namespace.
Will fix it in the separate PR with adding nGraph documentation, or in case of other major feedbacks will fix it in this PR.